sklearn/tests/test_multiclass.py

```diff
@@ -29,10 +29,10 @@
                                   SGDClassifier)
 from sklearn.tree import DecisionTreeClassifier, DecisionTreeRegressor
 from sklearn.model_selection import GridSearchCV, cross_val_score
-from sklearn.pipeline import Pipeline
+from sklearn.pipeline import Pipeline, make_pipeline
 from sklearn import svm
 from sklearn import datasets
-
+from sklearn.impute import SimpleImputer
```

Comment:
if meaningful, class it in alphabetic order

Reply:
do you want me to reorder all the import based on the class name?
I think it will lead to change in the ordering of all import statements is it fine?

Reply:
If it is not, let it as it is.

Reply:
okay, cool.